.travis.yml

```diff
@@ -5,11 +5,15 @@ os:
 before_install:
   - brew update
   - brew upgrade python
-  - pip3 install certifi dulwich pyqt5 urllib3
+  - pip3 install certifi dulwich pyqt5 tox urllib3
 
 install:
+  - pip3 install -e .
```

Comment:
This is not necessary anymore when using tox. Tox creates a Pext install in a virtualenv automatically.